docs/code-examples/line_strip2d_batch.py

```diff
@@ -18,4 +18,4 @@
 )
 
 # Log an extra rect to set the view bounds
-rr.log_rect("bounds", [3, 1.5, 8, 9], rect_format=rr.RectFormat.XCYCWH)
+rr2.log("bounds", rr2.Boxes2D([4.0, 4.5], centers=[3, 1.5]))
```

Comment:
I think we should try to find an API where we are more explicit here. It is not at all clear from reading this code what the first argument is

```suggestion
rr2.log("bounds", rr2.Boxes2D(half_size=[4.0, 4.5], centers=[3, 1.5]))
```